src/retroshare/rsposted.h

```diff
@@ -213,16 +279,51 @@ class RsPosted : public RsGxsIfaceHelper, public RsGxsCommentService
 //virtual bool createNewComment(uint32_t &token, RsGxsComment &comment) = 0;
 //virtual bool createNewVote(uint32_t &token, RsGxsVote &vote) = 0;
 
+	/**
+	 * @brief toggle message read status
+	 * @deprecated
+	 * @param[out] token GXS token queue token
+	 * @param[in] msgId
+	 * @param[in] read
+	 */
     RS_DEPRECATED_FOR(setPostReadStatus)
     virtual void setMessageReadStatus(uint32_t& token, const RsGxsGrpMsgIdPair& msgId, bool read) = 0;
         //////////////////////////////////////////////////////////////////////////////
-
-virtual bool createGroup(uint32_t &token, RsPostedGroup &group) = 0;
-virtual bool createPost(uint32_t &token, RsPostedPost &post) = 0;
-
-virtual bool updateGroup(uint32_t &token, RsPostedGroup &group) = 0;
-
-    virtual bool groupShareKeys(const RsGxsGroupId& group,const std::set<RsPeerId>& peers) = 0 ;
+	/**
+	 * @brief Request board creation.
+	 * The action is performed asyncronously, so it could fail in a subsequent
+	 * phase even after returning true.
+	 * @deprecated
+	 * @param[out] token Storage for RsTokenService token to track request
+	 * status.
+	 * @param[in] group Board data (name, description...)
+	 * @return false on error, true otherwise
+	 */
+	RS_DEPRECATED_FOR(createBoard)
+	virtual bool createGroup(uint32_t &token, RsPostedGroup &group) = 0;
+
+	/**
+	 * @brief Create board post. Blocking API.
+	 * @jsonapi{development}
+	 * @param[inout] post

```

Comment:
why is this `[inout]`?
Only change i can see is the removal of the image in `RsGxsPostedPostItem::fromPostedPost`

Reply:
i saw only example from channels wasnt sure there how is it right?
im not expert these things

```
	/**
	 * @brief Create channel post. Blocking API.
	 * @jsonapi{development}
	 * @deprecated
	 * @param[inout] post
	 * @return false on error, true otherwise
	 */
	RS_DEPRECATED_FOR(createPostV2)
	virtual bool createPost(RsGxsChannelPost& post) = 0;

	/**
	 * @brief Request post creation.
	 * The action is performed asyncronously, so it could fail in a subsequent
	 * phase even after returning true.
	 * @deprecated
	 * @param[out] token Storage for RsTokenService token to track request
	 * status.
	 * @param[in] post
	 * @return false on error, true otherwise
	 */
	RS_DEPRECATED_FOR(createPostV2)
	virtual bool createPost(uint32_t& token, RsGxsChannelPost& post) = 0;
```

Reply:
or it needs looks like this:

```
	/**
	 * @brief Request post creation.
	 * The action is performed asyncronously, so it could fail in a subsequent
	 * phase even after returning true.
	 * @param[out] token Storage for RsTokenService token to track request
	 * status.
	 * @param[in] post
	 * @return false on error, true otherwise
	 */
	virtual bool createPost(uint32_t& token, RsPostedPost& post) = 0;
```

Reply:
This looks good:
```c++
	/**
	 * @brief Request post creation.
	 * The action is performed asyncronously, so it could fail in a subsequent
	 * phase even after returning true.
	 * @param[out] token Storage for RsTokenService token to track request
	 * status.
	 * @param[in] post
	 * @return false on error, true otherwise
	 */
	virtual bool createPost(uint32_t& token, RsPostedPost& post) = 0;
```

Reply:
sorry, we both missed `@jsonapi{development}`

Reply:
yes i forget